### ISSUE TITLE
Poor Bridging Infill Anchor Points at Start/Stop of Path

### ISSUE THREAD

**Issue by lordofhyphens** [alexrj/Slic3r#4018]
### Version 
1.2.9, also 1.3.0-dev as of 9f0335b 

### Operating system type + version
Windows 8

### Behavior
I will say that the infill Overlap setting greatly improves bridging when connecting to the vertical walls but there seems to be a gap problem when the walls are not parallel to the X or Y axis where the bridge doesn't reach the wall at all.

![image](https://user-images.githubusercontent.com/31754/26910839-f8539426-4bce-11e7-8e4c-1f6e9492fa94.png)

The anchor points are not very effective for the beginning and end of bridging infill at 0% infill.

#### STL/Config (.ZIP) where problem occurs
https://github.com/alexrj/Slic3r/files/1059803/4012.zip

(Submitted for @JanComputerman as the result of splitting #4012 into different issues. @JanComputerman please subscribe to this issue)

**Comment by JanComputerman** [comment]
All extrudes at 0.5mm 50% infill overlap and 1.3.dev slic3r which crashes when trying to connect to my PrintrBot Jr so I exported the gcode and here is that infamous layer ...
![img_20170608_153545](https://user-images.githubusercontent.com/29179112/26947581-3a16d98a-4c61-11e7-8887-8de4ac491768.jpg)

**Comment by JanComputerman** [comment]
The edge where it fails to connect is at an angle to the x axis and also at 45 degrees from vertical 3 x 0.5mm perimeters wide on top so it is also possible that the "vertical" wall may not be exactly where it is supposed to be causing the failure to connect. The layer is printed right to left and joins just fine as the narrow section is reached. It fails again as it reaches the other end with the same shaped opening.

**Comment by dobrzann** [comment]
I have the same problem.
nozzle 0.4mm
layer 0.2mm
![20170617_125829](https://user-images.githubusercontent.com/25279164/27252392-d929fa22-535d-11e7-9b87-d47b0d2a8339.jpg)
![20170617_125927](https://user-images.githubusercontent.com/25279164/27252393-d92d2512-535d-11e7-8f6b-5c6c9c8c32e2.jpg)
![20170617_125234](https://user-images.githubusercontent.com/25279164/27252394-d92eb094-535d-11e7-904e-a37f3b59d0cf.jpg)
![20170617_125219](https://user-images.githubusercontent.com/25279164/27252395-d9303842-535d-11e7-852d-50d5881c8136.jpg)

Model:
[50x50 trojkat.zip](https://github.com/alexrj/Slic3r/files/1082501/50x50.trojkat.zip)

**Comment by dobrzann** [comment]
It looks better after I changed OVERLAP_FACTOR in flow.hpp form 1 to 2. 
Of course except walls and other perimeters
![20170617_140549](https://user-images.githubusercontent.com/25279164/27252778-b24eb4ca-5366-11e7-8ac3-da73896f2379.jpg)

After that change infill looks better too.

**Comment by JanComputerman** [comment]
Yeah that lower green image is classic zigzag fill no where near the wall like you have play in your drive and it is not going all the way to the wall ... seen that when it tried to fill a long thin gap and the extruder just buzzes in the Y direction as it slowly moves the X down the slot gap. Is there a gcode editor that could let you select a number of points and just move them a little using a graphical interface? Then we could make little adjustments to these little gap problems.

**Comment by JanComputerman** [comment]
Ok I tried the 6/24 release of Slic3r and it now seems to default to 55% Overlap and if set to 50% will drop the first 2 or 3 bridges at the narrow end where it begins to print the top layer. Another odd thing however is that it prints the layers (2 perimeters) at a modest speed until the layer prior to bridging and then will print that perimeter at a very high speed guessing 60mm/second ... it just whips around the top then bridges.
